src/ecmult_impl.h

```diff
@@ -82,7 +82,7 @@
  *  contain prej[0].z / a.z. The other zr[i] values = prej[i].z / prej[i-1].z.
```

Comment:
In commit c531ff8fb96cdaa871e637631a346bb1352cca3d:

The documentation above secp256k1_ecmult_odd_multiples_table is now outdated (it is only fixed in the next commit, I believe).

Reply:
It's probably because I don't know how to GitHub, but I intended for these comments to end up on #899 instead.

Reply:
Done.